spring-integration-file/src/main/java/org/springframework/integration/file/FileReadingMessageSource.java

```diff
@@ -71,12 +70,7 @@
 
 	private static final Log logger = LogFactory.getLog(FileReadingMessageSource.class);
 
-
-	private volatile File directory;
-
-	private volatile DirectoryScanner scanner = new DefaultDirectoryScanner();
-
-	private volatile boolean autoCreateDirectory = true;
+	private final static ThreadLocal<FileMessageHolder> RESOURCES = new ThreadLocal<FileMessageHolder>();
```

Comment:
This (and `FileMessageHolder`) should have been removed by INT-2685 - it is no longer used.